pkg/cmd/connectors/cluster/create/create.go

```diff
@@ -0,0 +1,95 @@
+package create
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	connectormgmtclient "github.com/redhat-developer/app-services-sdk-go/connectormgmt/apiv1/client"
+
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	name string
+
+	outputFormat string
+	f            *factory.Factory
+	interactive  bool
+}
+
+func NewCreateCommand(f *factory.Factory) *cobra.Command {
+	opts := &options{
+		f: f,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "create",
+		Short:   f.Localizer.MustLocalize("connector.cluster.create.cmd.shortDescription"),
+		Long:    f.Localizer.MustLocalize("connector.cluster.create.cmd.longDescription"),
+		Example: f.Localizer.MustLocalize("connector.cluster.create.cmd.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.name != "" {
+				// TODO
+			}
+
+			if !f.IOStreams.CanPrompt() && opts.name == "" {
+				return f.Localizer.MustLocalizeError("connector.common.error.requiredWhenNonInteractive")
+			} else if opts.name == "" {
+				opts.interactive = true
+			}
+
+			validOutputFormats := flagutil.ValidOutputFormats
+			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, validOutputFormats...) {
+				return flagutil.InvalidValueError("output", opts.outputFormat, validOutputFormats...)
+			}
+
+			return runCreate(opts)
+		},
+	}
+	flags := flagutil.NewFlagSet(cmd, f.Localizer)
+	flags.StringVar(&opts.name, "name", "", f.Localizer.MustLocalize("connectors.common.id.flag"))
+	flags.AddOutput(&opts.outputFormat)
+
+	return cmd
+}
+
+func runCreate(opts *options) error {
+	f := opts.f
+
+	var conn connection.Connection
+	conn, err := f.Connection(connection.DefaultConfigSkipMasAuth)
+	if err != nil {
+		return err
+	}
+
+	if opts.interactive {
```

Comment:
The current implementation already reads file from stdin if file is not specified, will the interactive mode need more stuffs to handle as well?

Reply:
Ah.. remove it then. No need

Reply:
Removing interactive from describe abd delete too

Reply:
Yes. I did not put much though into feasibility of interactive mode